### DISCUSSION TITLE
Linking r2cloud properly to my satnogs account

### DISCUSSION THREAD

**Discussion by lalliexperience** [dernasherbrezon/r2cloud#143] [Q&A] [answered]
I've enabled the satnogs feature in r2cloud but am I understanding correctly that this is satnogs satellite info input from Satnogs (and not output to satnogs)?  I would like to have the data decoded sent out to Satnogs but I don't see where I can put my Satnogs API key in r2cloud or leosatdata.com.  My satnogs account says they are not receiving data.

**Comment by dernasherbrezon** [comment]
Hi,

> am I understanding correctly that this is satnogs satellite info input from Satnogs (and not output to satnogs)?

Yes. r2cloud will download only the list of satellites at that point.

> data decoded sent out to Satnogs but I don't see where I can put my Satnogs API key in r2cloud or leosatdata.com

It's under "Intergrations" menu in leosatdata.com

<img width="1062" alt="Screenshot 2024-08-16 at 18 49 33" src="https://github.com/user-attachments/assets/f68f8eb8-bf7e-40cf-a80e-573ede316161">

The integration is based on SIDS protocol, because SATNOGS doesn't accept any third party observations. If you configured your callsign in the user's settings in leosatdata.com, then this callsign and the base station's grid id will be used. 

<img width="579" alt="Screenshot 2024-08-16 at 18 53 14" src="https://github.com/user-attachments/assets/8544796d-5fb8-4f2c-a13a-61b470c74b13">

**Comment by lalliexperience** [comment]
I have already completed these steps but was not seeing it in my satnogs account where I can configure my station and antenna types, etc.
![image](https://github.com/user-attachments/assets/907239be-618e-414d-b628-9e94232f8fa0)

**Reply by dernasherbrezon** [reply, marked as answer]
This is satnogs station configuration. It is not compatible with r2cloud station configuration. They even operate differently. Satnogs station receive observation requests from servers, while r2cloud start observations on its own and share results. 

If you just want to share results with satnogs, then everything should work out-of-the-box. No need to configure satnogs station.

**Comment by lalliexperience** [comment]
Does this mean it would not show up as a site on the network page:
https://network.satnogs.org/stations/

but then data would will show up on the telemetry pages?:
https://db.satnogs.org/

**Reply by dernasherbrezon** [reply]
Yes. You're right.
